docs/development/guides/src/produce_files.py

```diff
@@ -0,0 +1,11 @@
+import os
+import tarfile
+from quantum_serverless import save_result
+
+with open("./my_file.txt", "w") as f:
+    f.write("Hello!")
+
+with tarfile.open("/data/my_file.tar", "w:gz") as tar:
```

Comment:
The file in the data directory is not deleted after this execution.  The next execution of the same program (or generating the same name file) override the file. It it fails to generate the file, the old file may be down loaded.   It is user responsibility to generate an unique file name so that it would not be override or confused?

Reply:
yes, it is up to user to handle their files and names of them inside the program.